packages/client/src/widgets/Game/ui/Game/models/gameAudio.ts

```diff
@@ -0,0 +1,34 @@
+import { IGameAudio } from '../GameInterfaces';
+
+type LoadAudioFiles = (audioPaths: string[]) => Promise<HTMLAudioElement[]>;
+
+export const loadAudioFiles: LoadAudioFiles = (audioPaths) => {
+  const audioElements: HTMLAudioElement[] = audioPaths.map((path) => {
+    const audio = new Audio(path);
+    audio.preload = 'auto';
+    return audio;
+  });
+
+  const loadPromises: Promise<HTMLAudioElement>[] = audioElements.map((audio) => {
```

Comment:
Текущее название больше подходит функции
```suggestion
  const loadingPromises: Promise<HTMLAudioElement>[] = audioElements.map((audio) => {
```

Reply:
Согласна) тоже поменяла